.gitlab-ci.yml

```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
     - master
     - frankfurt
   script:
-    - kubectl_1.13 apply -f frankfurt/nucleus-dev
-    - kubectl_1.13 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
-    - kubectl_1.13 apply -f frankfurt/nucleus-stage
+    - kubectl_1.11 apply -f frankfurt/nucleus-dev
+    - kubectl_1.11 rollout status -f frankfurt/nucleus-dev/deploy.yaml -w --timeout=10m
+
+deploy frankfurt stage:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
+    - frankfurt
```

Comment:
I'm guessing having this branch update dev, stage, and prod is temporary for testing?

Reply:
The idea is that a push to the frankfurt branch would trigger parallel `kubectl apply` jobs across all the namespaces in the frankfurt cluster, but not (for example) any deployments in `iowa-b`.